ospf6d/ospf6_auth_trailer.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2021 Abhinay Ramesh
+ *
+ * This file is part of GNU Zebra.
+ *
+ * GNU Zebra is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * GNU Zebra is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; see the file COPYING; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#ifndef __OSPF6_AUTH_TRAILER_H__
+#define __OSPF6_AUTH_TRAILER_H__
+
+#include "lib/keychain.h"
+#include "ospf6_message.h"
+
+#define OSPF6_AUTH_HDR_MIN_SIZE 16
+#define OSPF6_AUTH_HDR_FULL     KEYCHAIN_MAX_HASH_SIZE + OSPF6_AUTH_HDR_MIN_SIZE
+
+#define OSPF6_AUTHENTICATION_NULL                 0
+#define OSPF6_AUTHENTICATION_CRYPTOGRAPHIC        1
+static const uint16_t CPID = 1;
+
+#define OSPF6_AUTH_SEQ_NUM_FILE "/etc/frr/ospf6_at.bin"
```

Comment:
This is the wrong location, it needs to be under `/var` and must be adjusted with `./configure` options.

Please look at how `ospf_gr_nvm_filepath` and `OSPFD_GR_STATE` are done.

Reply:
Have done changes to use ./configure to create file.